_artifacts/document-attributes-global.adoc

```diff
@@ -62,7 +62,7 @@ endif::[]
 
 // generic entities
 :org-name: Red{nbsp}Hat
-:rhodsdocshome: https://access.redhat.com/documentation/en-us/{url-productname-long}/{vernum}/
+:rhoaidocshome: https://access.redhat.com/documentation/en-us/{url-productname-long}/{vernum}/
```

Comment:
I noticed line 66 now contains the same attribute and same value as line 65. I don't think we need to have both, unless I'm missing something?